docs/docs/health-checks.md

```diff
@@ -58,6 +58,9 @@ OR
 
 #### Health check options
 
+* `command`: Command to run in order to determine the health of a task.
```

Comment:
You have to nest a `"value"` attribute inside. Maybe a link to the section in the rest-api.md would be nice.